hapless/main.py

```diff
@@ -340,3 +357,17 @@ def signal(self, hap: Hap, sig: signal.Signals):
                 f"{config.ICON_INFO} Cannot send signal to the inactive hap",
                 style=f"{config.COLOR_ERROR} bold",
             )
+
+    def restart(self, hap: Hap):
+        hid, name, cmd = hap.hid, hap.name, hap.cmd
```

Comment:
maybe name should contain `-restarted` suffix or something to have some hint for the user? wdyt

Reply:
Hmm, I would maybe personally prefer it without a suffix. If needed, some flag like `--suffix` or `--no-suffix` can be added.